Assets/Scripts/Model/Project.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.IO;
+using StereoApp.Model.Interfaces;
+using UnityEngine;
+
+namespace StereoApp.Model
+{
+    public class Project : ISerializableTo<Project, SerializedProject>
+    {
+        public static readonly string ProjectsDir = Path.Join(
+            Application.persistentDataPath,
+            "projects"
+        );
+
+        public SolidFigure Figure { get; set; }
+        public string ProjectName { get; set; }
+
+        public Project(SolidFigure figure)
+        {
+            Figure = figure;
+        }
+
+        public SerializedProject ToSerializable()
+        {
+            return new SerializedProject { figure = Figure.ToSerializable() };
+        }
+
+        public static Project FromProjectName(string projectName)
+        {
+            var projectPath = Path.Join(ProjectsDir, projectName);
+            using var fs = new StreamReader(projectPath);
+            var project = JsonSerializable.FromJson<Project, SerializedProject>(fs.ReadToEnd());
+            project.ProjectName = projectName;
+            return project;
+        }
+
+        public void Save()
```

Comment:
Do you know if it works both on PC and Android?

Reply:
It's supposed to, yes. I don't believe I tested it though.

Reply:
Yeah lets just merge it and wory about it later

Reply:
i.e. I'm using persistentDataPath which points to application's data folder and using standard lib which should bind to appropriate Android APIs but we'll see :)